package.json

```diff
@@ -54,8 +57,14 @@
     },
     "npm": false
   },
+  "pnpm": {
+    "peerDependencyRules": {
+      "ignoreMissing": ["@babel/*", "@eslint/*"]
+    }
+  },
   "volta": {
     "node": "16.18.1",
-    "yarn": "1.22.19"
+    "yarn": "1.22.19",
+    "pnpm": "7.33.5"
```

Comment:
Why an old version of pnpm?

Reply:
That's the version currently in ci. I figured updating it should be separate.